SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslationManager.cpp

```diff
@@ -3,15 +3,15 @@
 #include "EngineClasses/SpatialVirtualWorkerTranslationManager.h"
 #include "EngineClasses/SpatialVirtualWorkerTranslator.h"
 #include "Interop/Connection/SpatialWorkerConnection.h"
-#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialOSDispatcherInterface.h"
 #include "SpatialConstants.h"
 #include "Utils/SchemaUtils.h"
 
 DEFINE_LOG_CATEGORY(LogSpatialVirtualWorkerTranslationManager);
 
 SpatialVirtualWorkerTranslationManager::SpatialVirtualWorkerTranslationManager(
-	USpatialReceiver* InReceiver,
-	USpatialWorkerConnection* InConnection,
+	SpatialOSDispatcherInterface* InReceiver,
```

Comment:
I'm very open to a naming discussion on these things. Should we call them interfaces? "SpatialReceiver" is something everyone knows, but might not be the best name for the subset of the methods which are now part of the interface. Other thoughts?